AstSemantics.md

```diff
@@ -1,14 +1,19 @@
 # Abstract Syntax Tree Semantics
 
-WebAssembly code is represented as an Abstract Syntax Tree (AST) where each node
-represents an expression. Each function body consists of a list of expressions.
-All expressions and operators are typed, with no implicit conversions or overloading rules.
+This document describes WebAssembly semantics. The description here is written
+in terms of an Abstract Syntax Tree (AST), however it is also possible to
+understand WebAssembly semantics in terms of a typed stack machine. In practice,
```

Comment:
"it is also possible to X" suggests X is the only alternative. is it?

instead, how about "however, it is also possible to understand WebAssembly semantics in other ways, for example, in terms of a stack machine".